plugins/inventory/gitlab_runners.py

```diff
@@ -95,7 +100,8 @@ def _populate(self):
             if self.get_option('filter'):
                 runners = gl.runners.all(scope=self.get_option('filter'))
             else:
-                runners = gl.runners.all()
+                runners = gl.runners.all(all=True)
+                version_added: 1.0.0
```

Comment:
```suggestion
```
It needs to be added for the `api_token` ini option, not randomly inside the code!